fuzz/asn1.c

```diff
@@ -217,13 +217,14 @@ static ASN1_PCTX *pctx;
     if (type != NULL) { \
         int len2; \
         BIO *bio = BIO_new(BIO_s_null()); \
-        \
-        PRINT(bio, type); \
-        BIO_free(bio); \
-        len2 = I2D(type, &der); \
-        if (len2 != 0) {} \
-        OPENSSL_free(der); \
-        TYPE ## _free(type); \
+        if (bio != NULL) { \
+            PRINT(bio, type); \
```

Comment:
Only this needs to be inside the _if_ body.
Or this plus the BIO_free() call.

The rest need to be executed.

Reply:
Thanks, I have submit a fixup to correct all.